app/App_Resources/iOS/build.xcconfig

```diff
@@ -5,3 +5,4 @@
 // DEVELOPMENT_TEAM = YOUR_TEAM_ID;
 ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
+CODE_SIGN_ENTITLEMENTS = ;
```

Comment:
What does this do?

Reply:
it overwrites the default values after you create a new project in NativeScript

Reply:
Is this required in order to interact with the iOS keychain on the iOS simulator?